docker-compose.env

```diff
@@ -14,12 +14,11 @@ POSTGRES_PASSWORD=test
 
 ######### L2 GETH VARS #########
 TARGET_GAS_LIMIT
-VOLUME_PATH=/mnt/l2-node/l2
+VOLUME_PATH=/l2-node/l2
```

Comment:
`/mnt` is mounted into the root of the gitrepo, so we need to change this to prevent the geth db from being left behind in the host filesystem outside of the docker volume